config/test.js

```diff
@@ -25,4 +25,12 @@ module.exports = {
     filename: './test-db.sqlite3',
     removeBeforeOpen: true,
   },
+  authentication: {
+    enabled: false,
+    defaultUser:  {              // DO NOT USE THIS IN PRODUCTION
```

Comment:
Since this is in test.js it will only ever get used when NODE_ENV is set to test.